src/components/forms/cbr/CbrZoneExclusionAddressForm.js

```diff
@@ -0,0 +1,160 @@
+            hideHelperText={true}
+            className="fieldWidthSmaller"
+            {...cbrInvalid("account_id", this.state.account_id)}
+          />
+          <IcseTextInput
+            id={this.props.data.name + "-cbr-zone-location"}
+            componentName={this.props.data.name + "cbr-zone-location"}
+            className={"fieldWidthSmaller"}
+            labelText={"Location"}
+            field="location"
+            value={this.state.location}
+            onChange={this.handleInputChange}
+            hideHelperText={true}
+            {...cbrInvalid("location", this.state.location)}
+          />
+        </IcseFormGroup>
+        <IcseFormGroup>
+          <IcseTextInput
+            id={this.props.data.name + "-cbr-zone-service-name"}
+            componentName={this.props.data.name + "cbr-zone-service-name"}
+            className={"fieldWidthSmaller"}
+            labelText={"Service Name"}
+            field="service_name"
+            value={this.state.service_name}
+            onChange={this.handleInputChange}
+            hideHelperText={true}
+            {...cbrInvalid("service_name", this.state.service_name)}
+          />
+          <IcseTextInput
+            id={this.props.data.name + "-cbr-zone-service-type"}
+            componentName={this.props.data.name + "cbr-zone-service-type"}
+            className={"fieldWidthSmaller"}
+            labelText={"Service Type"}
+            field="service_type"
+            value={this.state.service_type}
+            onChange={this.handleInputChange}
+            hideHelperText={true}
+            {...cbrInvalid("service_type", this.state.service_type)}
+          />
+        </IcseFormGroup>
+        <IcseFormGroup>
+          <IcseSelect
+            labelText="Type"
+            name="type"
+            formName={this.props.data.name + "cbr-zone-type"}
+            groups={["IP Address", "IP Range", "Subnet", "VPC", "Service Ref"]}
+            value={typeNameMap[this.state.type]}
+            handleInputChange={this.handleInputChange}
+            invalidText="Select a Type"
+            className="fieldWidthSmaller"
+          />
+          <IcseTextInput
+            id={this.props.data.name + "-cbr-zone-value"}
+            componentName={this.props.data.name + "cbr-zone-value"}
+            className={"fieldWidthSmaller"}
+            labelText={"Value"}
+            field="value"
+            value={this.state.value}
+            onChange={this.handleInputChange}
+            invalid={false}
+            hideHelperText={true}
+          />
+        </IcseFormGroup>
+      </>
+    );
+  }
+}
+
+CbrZoneExclusionAddressForm.defaultProps = {
+  data: {
+    name: "",
+    account_id: "",
+    location: "",
+    service_name: "",
+    service_type: "",
+    type: "ipAddress",
+    value: "",
+  },
+  isModal: false,
+};
+
+CbrZoneExclusionAddressForm.propTypes = {
+  data: PropTypes.shape({
+    account_id: PropTypes.string.isRequired,
```

Comment:
issue description says that all these fields should be optional. also, it looks like we're missing _service instance_?